www/static/deno_lint.js

```diff
@@ -0,0 +1,240 @@
+let wasm;
```

Comment:
Is there a way we could have this js file, the wasm file, and the .d.ts file built when creating the website? Then we don't need to keep updating these and they don't bloat the repository.

Reply:
Yeah I was thinking about that, and put it off for the moment to focus on the implementation part. I'll try to find a way to achieve continuous delivery.

Reply:
Thinking about it a little bit further, I'd prefer to have developers commit the wasm files into the repo instead of relying on CI to build them, mostly because building them in CI would require configuration changes in Deno Deploy. Of course we could do that, but I think we can do so later if it's really needed (for instance we may want to do AoT build for fresh, which can be done together with wasm build in CI)